mbf_costmap_nav/src/nav_core_wrapper/wrapper_global_planner.cpp

```diff
@@ -50,11 +50,13 @@ uint32_t WrapperGlobalPlanner::makePlan(const geometry_msgs::PoseStamped &start,
                                         double &cost,
                                         std::string &message)
 {
-#ifdef USE_OLD_NAV_CORE
+#if ROS_VERSION_MINIMUM(1, 10, 0) // if current ros version is >= 1.10.0
```

Comment:
indigo is minor version 11 and jade (which should have been 11) is not even in the list anymore (maybe messed up somehow) so to support the LTS versions this has to be `ROS_VERSION_MINIMUM(1,12,0)`.

Reply:
right! thanks